### DISCUSSION TITLE
Execute a DB call within the scheduled job

### DISCUSSION THREAD

**Discussion by divyagangadhar** [quartznet/quartznet#1146] [Q&A]
**Is your feature request related to a problem? Please describe.**

**Describe the solution you'd like**

**Describe alternatives you've considered**

**Additional context**
![image](https://user-images.githubusercontent.com/38987776/112976460-86468580-9172-11eb-90f4-471356cd26fe.png)

How can I get my job execute DB call?
I've got the following job that I want to execute in the given image
Unfortunately it never works and if I remove the DB call everything works perfect

The image to my job scheduler :
![image](https://user-images.githubusercontent.com/38987776/112976918-11c01680-9173-11eb-8d8d-e422a45d0894.png)

**Comment by lahma** [comment]
What is the error you get?

**Reply by divyagangadhar** [reply]
I used a custom job factory and it gave me this exception:
: 'No service for type 'x.TestJob' has been registered.'

Test Job is the job I need to execute

**Reply by lahma** [reply]
Please see the example application how to integrate with MS DI: https://github.com/quartznet/quartznet/tree/main/src/Quartz.Examples.AspNetCore

**Reply by divyagangadhar** [reply]
Hey thanks for sharing the above URL. I have gone through it and I have implemented the job as per the above examples and it is working fine. 

But when I try adding dbContext by using dependency injection to my TestJob(as per the above screenshot), it is not hitting the TestJob

**Reply by lahma** [reply]
Your image does not look like the example code I gave you. Also check that you have registered the DbContext properly and that you job factory is producing scoped jobs.
